python/pyngraph/onnx_import.cpp

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "pyngraph/onnx_import.hpp"
+#include <istream>
+#include <memory>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <string>
+#include <vector>
+#include "ngraph/frontend/onnx_import/onnx.hpp"
+#include "ngraph/function.hpp"
+
+namespace py = pybind11;
+
+static std::vector<std::shared_ptr<ngraph::Function>>
+    load_onnx_model(const std::string& model_proto)
+{
+    std::istringstream iss(model_proto,
+                           std::ios_base::binary | std::ios_base::in | std::ios_base::ate);
+    return ngraph::onnx_import::load_onnx_model(iss);

```

Comment:
Why `std::ios_base::ate`? This means seek to the end.

Reply:
My mistake, actually it's not necessary.